apps/storybook/.eslintrc.json

```
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    "next/core-web-vitals",
+    "next/typescript",
+    "plugin:storybook/recommended"
+  ]
+}
```

```suggestion
{
  "extends": [
    "next/core-web-vitals",
    "next/typescript",
    "plugin:storybook/recommended",
    "@repo/eslint-config/next.js"
  ]
}
```
사소하긴 한데 storybook 컴포넌트 작성에서도 eslint 규칙을 따를 수 있게 추가하는건 어떨까?